### PR TITLE
[WNMGDS-632] Apply Review classes

### PR DESCRIPTION
**Pull request by nichia**
This fixes the issue of Review component classes cannot be overridden because the component applies utility styles which uses `!important`.

### Changed
- Update Review component to apply it's component styling instead of using utility classes.

### Added 
- `ds-c-review` and `ds-c-review__heading` classes 

### How to test
- Run the design system site locally
- Update Review example to [override classes](https://github.com/CMSgov/design-system/issues/817) and check that the styles applied correctly

### CODE REVIEW

**Review by line47 — APPROVED** [review]
This looks good to me

File: packages/design-system/src/components/Review/Review.jsx [hunk 1]
```diff
@@ -25,10 +21,7 @@ export class Review extends React.PureComponent {
       onEditClick,
       editContent,
     } = this.props;
-    const classes = classNames(
-      'ds-c-review ds-u-border-bottom--2 ds-u-padding-y--2 ds-u-justify-content--between ds-u-display--flex',
-      className && className
-    );
+    const classes = classNames('ds-c-review', className && className);
```
Comment (bernardwang) [inline packages/design-system/src/components/Review/Review.jsx:24]: I'm actually not sure if the check on `className` is necessary. `classNames()` might actually handle that automatically, @nichia can you test and make sure?

**Comment by bernardwang** [comment]
We also need to update the `Review` html examples

File: packages/design-system-docs/src/pages/components/Review/review.example.html [hunk 2]
```diff
@@ -1,8 +1,6 @@
-<div
-  class="ds-u-border-bottom--2 ds-u-padding-y--2 ds-u-justify-content--between ds-u-display--flex"
->
+<div class="ds-c-review">
   <div class="ds-u-margin-right--2">
-    <h3 class="ds-text ds-u-margin-bottom--0 ds-u-font-weight--bold ds-u-display--inline-block">
+    <h3 class="ds-c-review__heading ds-text">
```
Comment (bernardwang) [inline packages/design-system-docs/src/pages/components/Review/review.example.html:3]: What is this additional `ds-text` for?

Comment (nichia) [reply packages/design-system-docs/src/pages/components/Review/review.example.html]: Thanks for picking this up. Have updated.

File: packages/design-system/src/styles/components/_Review.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  border-bottom: 2px solid $border-color;
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 16px;
+  padding-top: 16px;
+}
+
+.ds-c-review__heading {
+  display: inline-block;
+  font-weight: $font-bold;
```
Comment (line47) [inline packages/design-system/src/styles/components/_Review.scss:13]: I think this font size needs to be added. Take a look at this local branch and compare to production and you will see the production header is smaller than locally. 

```suggestion
  font-weight: $font-bold;
  font-size: $base-font-size;
```

**Review by line47 — APPROVED** [review]
After the font-size is added this looks good to me

File: packages/design-system/src/styles/components/_Review.scss (hunk 3 above)
Comment (bernardwang) [reply packages/design-system/src/styles/components/_Review.scss:13]: nice catch!

File: packages/design-system/src/styles/components/_Review.scss [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  font-size: $base-font-size;
+  margin-bottom: 0;
+  margin-top: 0;
```
Comment (bernardwang) [inline packages/design-system/src/styles/components/_Review.scss:16]: does this margin-top need to be here too? I dont see that originally in the utility classes

Comment (bernardwang) [reply packages/design-system/src/styles/components/_Review.scss:16]: oh i see is this from the `ds-text`?

**Review by bernardwang — APPROVED** [review]
LGTM!

File: packages/design-system/src/styles/components/_Review.scss (hunk 4 above)
Comment (nichia) [reply packages/design-system/src/styles/components/_Review.scss:16]: It's from `ds-u-margin-bottom--0`

**Pull request merged**